modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/tabletoolbar/plugin.js

```diff
@@ -954,7 +954,9 @@
 
 				let panelBlock;
 
-				panel.className = `${panel.className ?? ''} cke_combopanel`;
+				panel.className = panel.className
+					? `${panel.className} lfr-cke_table_panel`
+					: 'lfr-cke_table_panel';
```

Comment:
It looks like `panel.className` is always undefined at this point, with class inserting logic done later. So, we can simplify to:
```js
panel.className = 'lfr-table-panel'
```

I renamed the class there for consistency:
- `cke` prefix is used by ckeditor. Our classes should not contain it.
- our classes should have only dashes (`-`)

Reply:
> It looks like panel.className is always undefined at this point, with class inserting logic done later. So, we can simplify to:

I was thinking if someone could place a className in panel definition but it's a premature optimization. Removing!

> cke prefix is used by ckeditor. Our classes should not contain it.
> our classes should have only dashes (-)

Oh, I didn't know it. Changing! :D